subsys/bluetooth/Kconfig

```diff
@@ -172,6 +172,13 @@ config BT_PER_ADV_SYNC_TRANSFER_RECEIVER
 config BT_PER_ADV_SYNC_TRANSFER_SENDER
 	bool "Periodic Advertising Sync Transfer sender"
 	depends on !BT_CTLR || BT_CTLR_SYNC_TRANSFER_SENDER_SUPPORT
+
+config BT_REQUEST_PEER_SCA
```

Comment:
Do you plan adding host code for this?

Reply:
Well, to be honest I'm a bit in doubt as to what is required for the host part? Is it 'just' to expose this to a bt_le_req_peer_sca() function?

Reply:
Thinking further - I think, I'd probably leave this out of this, and then let future decide if host support is needed for this.